mantle/cmd/kola/kola.go

```diff
@@ -173,6 +189,51 @@ func runRun(cmd *cobra.Command, args []string) error {
 		patterns = args
 	}
 
+	return kolaRunPatterns(patterns, runRerunFlag)
+}
+
+func runRerun(cmd *cobra.Command, args []string) error {
+	var patterns []string
+	type jsonTest struct {
+		Name   string                `json:"name"`
+		Result testresult.TestResult `json:"result"`
+		Output string                `json:"output"`
+	}
+	type jsonSuiteResult struct {
+		Tests  []jsonTest            `json:"tests"`
+		Result testresult.TestResult `json:"result"`
+
+		// Context variables
+		Platform string `json:"platform"`
+		Version  string `json:"version"`
+	}
+
+	file, err := os.Open(filepath.Join(kola.Options.CosaWorkdir, "tmp/kola/reports/report.json"))
+	if err != nil {
+		return err
+	}
+
+	bytes, _ := ioutil.ReadAll(file)
```

Comment:
Any reason not to propagate the error here?

Reply:
Nope, just an oversight on my part.